src/client/entity-editor/entity-editor.tsx

```diff
@@ -73,7 +72,7 @@ const EntityEditor = (props: Props) => {
 				<AliasEditor show={aliasEditorVisible} {...props}/>
 				<NameSection {...props}/>
 				<ButtonBar {...props}/>
-				<RelationshipSection {...props}/>
+				<RelationshipSection {...props as any}/>
```

Comment:
Is this necessary? I don't see TS complain when I remove it.

Also, the next file in the git diff (entity-merge.tsx) uses `<RelationshipSection {...(props as any)}/>` with the extra parentheses, not sure if that makes a difference.

Reply:
Looks fine to me too without it, I think it may have been left over from an earlier draft of the commit! Will remove.